src/test/java/com/andersen/UserServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.andersen;
+
+import com.andersen.bus.TicketType;
+import com.andersen.entity.User;
+import com.andersen.repo.UserRepository;
+import com.andersen.service.UserService;
+import com.andersen.service.UserServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class UserServiceTest {
+
+	@Mock
+	private UserRepository userRepository;
+
+	@InjectMocks
+	private UserServiceImpl userService;
+
+	private User user;
+
+	@BeforeEach
+	void setUp() {
+		user = new User();
+		user.setId(1);
+		user.setName("Bjorn");
+		user.setCreationDate(LocalDateTime.now());
+	}
+
+	@Test
+	void findUserById_Found() {
+
+		when(userRepository.findById(1)).thenReturn(Optional.of(user));
+		User result = userService.findById(1);
+
+		assertEquals(result, user);
+	}
+
+	@Disabled
```

Comment:
why this annotation?

Reply:
I just temporarily decided to disable some test methods

Reply:
Not the best solution in a real project)))) or if you need to, just add a comment as to why it's disabled